tests/test_record_resource.py

```diff
@@ -52,46 +52,47 @@ def test_create_read_record(client, input_record):
 # because the CI creates a new container for each test run. It will work
 # once locally, but fail on subsequent run.
 # TODO: FIX to make it clean up itself. Adding es_clear didn't work
-# def test_create_search_record(client, input_record):
-#     """Test record search."""
-#     # Search records, should return empty
-#     response = client.get("/records_v2", headers=HEADERS)
-#     assert response.status_code == 200
+@pytest.mark.skip(reason="Fix to make it clean up itself")
```

Comment:
@zzacharo's PR has a different approach. Be careful in merge order according to what you decide should prevail.

Reply:
Zach's approach should prevail